### ISSUE TITLE
Disable NOVA + Nutri-Score on OFF if category = Non food products

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#1577]

**Comment by VaiTon** [comment]
I think this has been done. @teolemon @stephanegigandet

**Comment by CharlesNepote** [comment]
It's not fixed, see https://world.openfoodfacts.org/category/non-food-products/nova-groups

**Comment by vanshikaarora** [comment]
Hey @teolemon @VaiTon if no one is working on this issue. Can I please take this up?

**Comment by stephanegigandet** [comment]
Hi @vanshikaarora, sure, go ahead. It has been fixed for NOVA, but not for the Nutri-Score 

Nova fix is at: https://github.com/openfoodfacts/openfoodfacts-server/blob/master/lib/ProductOpener/Food.pm#L4995

**Comment by VaiTon** [comment]
Hey @vanshikaarora sure you can! All contributions are well accepted here!
I would say good luck with it! 😆

**Comment by vanshikaarora** [comment]
Hey @VaiTon @stephanegigandet @CharlesNepote , how to check whether it is a non-food product, is there a specific category tag for non-food products?

**Comment by VaiTon** [comment]
Yep, see https://world.openfoodfacts.org/category/non-food-products

**Comment by vanshikaarora** [comment]
>Yep, see https://world.openfoodfacts.org/category/non-food-products

@VaiTon in the class `CategoryName` the field `name` would return `Non food products` right?Or is there a different feild with the value `Non food products`?

**Comment by VaiTon** [comment]
> > Yep, see https://world.openfoodfacts.org/category/non-food-products
> 
> @VaiTon in the class `CategoryName` the field `name` would return `Non food products` right?Or is there a different feild with the value `Non food products`?

I think that should be right but @stephanegigandet could definitely answer correctly.

**Comment by hangy** [comment]
I believe, it should be possible to check for that category with `has_tag($product_ref, 'categories', 'en:non-food-products')`.

**Comment by vanshikaarora** [comment]
Hey, I have solved this Issue here https://github.com/openfoodfacts/openfoodfacts-androidapp/pull/2686. Kindly review :)
